src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -123,3 +124,22 @@ def is_member_of(group_id: str, member_id: str) -> bool:
             CheckGroupMembershipParameters(group_id=group_id, member_id=member_id)
         ).value
     )
+
+
+def get_scaleset_idenity_resource_path() -> str:
```

Comment:
Is there documentation we can link to describing what path spec we are trying to satisfy? Maybe something in the managed identity docs?

Reply:
Generally: I think relating this code to the ARM template and what we're doing conceptually will be useful in the future.